protocols/floodsub/src/protocol.rs

```diff
@@ -95,15 +95,15 @@ where
 #[derive(Debug)]
 pub enum FloodsubDecodeError {
     /// Error when reading the packet from the socket.
-    ReadError(upgrade::ReadOneError),
+    ReadError(io::Error),
```

Comment:
This is a breaking change that needs to be mentioned in the changelog and needs a changelog entry!